### PR TITLE
refactor: Rework entry point scripts to run 'listenTcp' command as non-root

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/master/.github/Contributing.md.

## What is the current behavior?
'listenTcp' command runs as root everywhere it is used.

## What is the new behavior?
'listenTcp' command now runs as non-root everywhere it is used.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
the combination between exec and & doesn't fit well...

File: cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh [hunk 1]
```diff
@@ -70,7 +70,7 @@ echo "$(date) ENABLE_REGISTRY_ACL = ${ENABLE_REGISTRY_ACL}"
 
 if [ "${ENABLE_REGISTRY_ACL}" == "true" ]; then
   echo "$(date) Starting edgex-core-consul with ACL enabled ..."
-  docker-entrypoint.sh agent \
+  exec docker-entrypoint.sh agent \
```
Comment (jim-wang-intel) [inline cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh:73]: consul is running as a background process and `exec` doesn't make sense here

Comment (jim-wang-intel) [reply cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh:73]: exec should only be always the last one command as it replaces the current program in the current process, without forking a new process

File: cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh [hunk 2]
```diff
@@ -96,7 +96,7 @@ if [ "${ENABLE_REGISTRY_ACL}" == "true" ]; then
   # no need to wait for Consul's port since it is in ready state after all ACL stuff
 else
   echo "$(date) Starting edgex-core-consul with ACL disabled ..."
-  docker-entrypoint.sh agent \
+  exec docker-entrypoint.sh agent \
```
Comment (jim-wang-intel) [inline cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh:99]: remove exec

File: cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh (hunk 1 above)
Comment (lenny-intel) [reply cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh]: Yea, was playing with this and forgot to remove and didn't see the `&`. Removed. THX!

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**